kettle-plugins/hadoop-cluster/ui/src/main/javascript/app/components/creating/creating.component.js

```diff
@@ -29,9 +29,9 @@ define([
     controller: creatingController
   };
 
-  creatingController.$inject = ["$state", "$timeout", "$stateParams"];
+  creatingController.$inject = ["$state", "$timeout", "$stateParams", "dataService"];
```

Comment:
not in my area over expertise, but should this be $dataService?  just noticed that it's different from the other three.  same with the function right below

Reply:
without the $ is correct in this case.

Reply:
"dataService" is a parameter that injects a custom service. The dollar-sign is reserved for the default services of AngularJS.